pkgs/misc/vim-plugins/default.nix

```diff
@@ -19,8 +19,9 @@ let
 
 
 # TL;DR
-# Add your plugin to ./vim-plugin-names
-# Regenerate via `nix-shell -I nixpkgs=/path/to/your/local/fork -p vimPlugins.pluginnames2nix --command "vim-plugin-names-to-nix +'w result' +qall"`
+# * Add your plugin to ./vim-plugin-names
+#   * sort -df ./vim-plugin-names > sorted && mv sorted vim-plugin-names
+# * Regenerate via `nix-shell -I nixpkgs=/path/to/your/local/fork -p vimPlugins.pluginnames2nix --command "vim-plugin-names-to-nix +silent +'x! result'"`
```

Comment:
Does that fix the pagination?

Reply:
The '+silent' did, for me anyway! :grin:

Reply:
Nice! We might be able to close https://github.com/NixOS/nixpkgs/issues/38748 then.